indico/modules/events/editing/client/js/management/editable_type/NextEditable.jsx

```diff
@@ -205,11 +205,11 @@ function NextEditableDisplay({eventId, editableType, onClose, fileTypes, managem
             filters={filters}
             filterOptions={filterOptions}
             searchableId={e => e.contributionFriendlyId}
-            searchableFields={e =>
-              e.editor
-                ? [e.contributionTitle, e.contributionCode, e.editor.fullName]
-                : [e.contributionTitle, e.contributionCode]
-            }
+            searchableFields={e => [
+              e.contributionTitle,
+              e.contributionCode,
+              e.contributionPersons.join(''),
```

Comment:
```suggestion
              e.contributionPersons.join('|'),
```

Reply:
just had a much better idea, i feel dumb now 😄 

```suggestion
              ...e.contributionPersons,
```